src/Util.ts

```diff
@@ -445,124 +445,133 @@ class Util extends BITBOXUtil {
-            const voutMatch = slpData.spendData.filter(
-              (x: any) => thisUtxo.vout === x.vout
-            )
-            // console.log(`voutMatch: ${JSON.stringify(voutMatch, null, 2)}`)
+            outAry[i] = thisUtxo
+          }
+        }
 
-            // If there are no vout matches, the UTXO can safely be marked as false.
-            if (voutMatch.length === 0) {
-              validations[i] = false
-            }
+        // Handle Send SLP transactions.
+        if (slpData.transactionType === "send") {
+          // Filter out any vouts that match.
+          const voutMatch = slpData.spendData.filter(
+            (x: any) => thisUtxo.vout === x.vout
+          )
+          // console.log(`voutMatch: ${JSON.stringify(voutMatch, null, 2)}`)
+
+          // If there are no vout matches, the UTXO can safely be marked as false.
+          if (voutMatch.length === 0) {
+            outAry[i] = false
+          }
 
-            // If UTXO passes validation, then return formatted token data.
-            else {
-              const genesisData = await this.decodeOpReturn(slpData.tokenId)
-              // console.log(
-              //   `genesisData: ${JSON.stringify(genesisData, null, 2)}`
-              // )
-
-              // Hydrate the UTXO object with information about the SLP token.
-              thisUtxo.utxoType = "token"
-              thisUtxo.tokenId = slpData.tokenId
-              thisUtxo.tokenTicker = genesisData.ticker
-              thisUtxo.tokenName = genesisData.name
-              thisUtxo.tokenDocumentUrl = genesisData.documentUrl
-              thisUtxo.tokenDocumentHash = genesisData.documentHash
-              thisUtxo.decimals = genesisData.decimals
-
-              // Calculate the real token quantity.
-              thisUtxo.tokenQty =
-                voutMatch[0].quantity / Math.pow(10, thisUtxo.decimals)
-
-              validations[i] = thisUtxo
-            }
+          // If UTXO passes validation, then return formatted token data.
+          else {
+            const genesisData = await this.decodeOpReturn(slpData.tokenId)
+            // console.log(
+            //   `genesisData: ${JSON.stringify(genesisData, null, 2)}`
+            // )
+
+            // Hydrate the UTXO object with information about the SLP token.
+            thisUtxo.utxoType = "token"
+            thisUtxo.tokenId = slpData.tokenId
+            thisUtxo.tokenTicker = genesisData.ticker
+            thisUtxo.tokenName = genesisData.name
+            thisUtxo.tokenDocumentUrl = genesisData.documentUrl
+            thisUtxo.tokenDocumentHash = genesisData.documentHash
+            thisUtxo.decimals = genesisData.decimals
+
+            // Calculate the real token quantity.
+            thisUtxo.tokenQty =
+              voutMatch[0].quantity / Math.pow(10, thisUtxo.decimals)
+
+            outAry[i] = thisUtxo
           }
         }
       }
 
-      return validations
+      return outAry
```

Comment:
I changed the output variable name from `validations` to `outAry`. That's confusing git diff, and making it appear that I changed much more code than I actually did.